manticore/models/query.py

```diff
@@ -57,3 +62,15 @@ def _match_expression(self) -> Match:
             where.add(match, AND)
 
         return match
+
+    def _check_field_models(self, fields):
+        """ Сhecks that the field is in the model """
+        index_fields = self.query.model._meta.get_fields()
+        index_field_dict = {f.name: f for f in index_fields}
+        self.query.names_to_path([*fields], self.query.get_meta())
```

Comment:
name_to_path is not necessary, because manticore does not support join, and lookup support is not needed either.

Reply:
I wanted to use to validate fields in the model. rolled back